Modules/Compatibility/Deprecated/include/itkTreeChangeEvent.h

```diff
@@ -130,7 +129,6 @@ class TreeNodeChangeEvent : public TreeChangeEvent<TTreeType>
     : TreeChangeEvent<TTreeType>(s)
   {}
 
-private:
   void
   operator=(const Self &) = delete;
```

Comment:
@mseng10 All public Constructors/Destructors/Assignment operators should all be co-located near the top of the class definition.   The operator= .... = delete should be immediately before the destructor definition.

Consistency in the placement of these key functions is of paramount importance for the maintenance of the codebase.

Reply:
@hjmjohnson I have gone through all the changes and have done some re-ordering. Let me know your thoughts.